app-i18n/man-pages-it/man-pages-it-4.00.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
I think the EAPI bump is actually better *after* the bump instead of modifying the old version that hasn't been touched since 2015.